Scripts/Python/plasma/Plasma.py

```diff
@@ -1367,6 +1375,10 @@ def isEnabled(self):
         """Is the audio enabled? Returns 1 if true otherwise returns 0."""
         pass
 
+    def areSubtitlesEnabled(self):
+        """Are subtitles for the audio enabled? Returns 1 if true otherwise returns 0."""
+        pass
+
```

Comment:
This needs to be alphabetized like the rest of the stubs.

Reply:
You're gonna have to explain a bit more for me to understand what you mean. The rest of the stubs in this file are camelCase just like this one, no?

Reply:
The method starts with `a` so it should be near the start of the method list in this class